native/Avalonia.Native/src/OSX/menu.mm

```diff
@@ -0,0 +1,179 @@
+#include "common.h"
+#include "IGetNative.h"
+
+class AvnAppMenuItem : public ComSingleObject<IAvnAppMenuItem, &IID_IAvnAppMenuItem>, public IGetNative
+{
+private:
+    NSMenuItem* _native;
+public:
+    FORWARD_IUNKNOWN()
+    
+    AvnAppMenuItem()
+    {
+            /*id appMenu = [NSMenu new];
+            [appMenu setTitle:@"MenuTitle"];
+            
+            id testMenuItem = [[NSMenuItem alloc] initWithTitle:@"Test" action:NULL keyEquivalent:@""];
+            [appMenu addItem:testMenuItem];
+            
+            
+            id appMenuItem = [NSMenuItem new];
+            [[NSApp mainMenu] addItem:appMenuItem];
+            
+            [appMenuItem setSubmenu:appMenu];*/
+        
+        
+        _native = [NSMenuItem new];
+    }
+    
+    void* GetNative() override
+    {
+        return (__bridge void*) _native;
+    }
+    
+    virtual HRESULT SetSubMenu (IAvnAppMenu* menu) override
+    {
+        auto nsMenu = (__bridge NSMenu*) dynamic_cast<IGetNative*>(menu)->GetNative();
```

Comment:
Bridging pointers is a bad idea since it's easy to break reference counting, return `NSMenu*` directly from the method